rotkehlchen/tests/unit/decoders/test_makerdao.py

```diff
@@ -57,7 +56,118 @@ def test_makerdao_simple_transaction(
             notes='Withdraw 0.6 ETH from ETH-A MakerDAO vault',
             counterparty=CPT_VAULT,
             address=string_to_evm_address('0x809aade1B623d8cDAeb86484d3366A03F8841FBc'),
-            identifier=None,
             extra_data={'vault_type': 'ETH-A'},
         ),
     ]
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0xa217BDa86b0EDb86eE7d4D6e34F493eDF1ea4F29']])  # noqa: E501
+def test_makerdao_withdraw_dai_from_sdai(
+        database,
+        ethereum_inquirer,
+):
+    tx_hash = deserialize_evm_tx_hash('0x6b2a1f836cfc7c28002e4ac60297daa6d79fcde892d9c3b9ca723dea2f21af5c')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp = TimestampMS(1695854591000)
+    assert events == [
+        EvmEvent(
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal('0.001301015216220134'), usd_value=ZERO),
+            location_label='0xa217BDa86b0EDb86eE7d4D6e34F493eDF1ea4F29',
+            notes='Burned 0.001301015216220134 ETH for gas',
+            tx_hash=tx_hash,
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.DEPOSIT_ASSET,
+            asset=A_SDAI,
+            balance=Balance(amount=FVal('16.020774067834506624'), usd_value=ZERO),
+            location_label='0xa217BDa86b0EDb86eE7d4D6e34F493eDF1ea4F29',
+            notes='Deposit 16.020774067834506624 sDAI to sDAI pot',
+            tx_hash=tx_hash,
+            counterparty=CPT_SDAI,
+            address=string_to_evm_address('0x83F20F44975D03b1b09e64809B757c47f942BEeA'),
+        ), EvmEvent(
+            sequence_index=2,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.REMOVE_ASSET,
+            asset=A_DAI,
+            balance=Balance(amount=FVal('16.601085935411927527'), usd_value=ZERO),
```

Comment:
Please don't make me repeat myself.

```suggestion
            balance=Balance(amount=FVal('16.601085935411927527'),
```